Saule/Http/PreprocessingDelegatingHandler.cs

```diff
@@ -62,6 +62,11 @@ public PreprocessingDelegatingHandler(JsonApiConfiguration config)
 
             var value = result.Content as ObjectContent;
 
+            if (value?.Value is Newtonsoft.Json.Linq.JObject)
```

Comment:
good question. I'll try removing the changes in the other file, and see what happens